web/static/js/components/stage_help.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import ReactDOM from "react-dom"
+
+import { connect } from "react-redux"
+import { bindActionCreators } from "redux"
+import { actions } from "../redux/retro"
+
+import * as AppPropTypes from "../prop_types"
+import stageConfig from "../configs/stage_configs"
+
+export const StageHelp = props => {
+  const handleClick = () => {
+    const { retro, actions: { showStageHelp } } = props
+    showStageHelp(retro)
+  }
+
+  const showIcon = () => {
+    const { retro: { stage } } = props
+    return (stageConfig[stage] && stageConfig[stage].help)
```

Comment:
we should be able to axe the short-circuiting here, as each `stage` is a constant, and are all accounted for in the `stageConfig` object. While in theory they _can_ change, any unaccounted for changes would result in a number of end-to-end tests failing, so we shouldn't need to be defensive here. unless i'm missing something?

Reply:
Sure - it's just such a common error I just do it reflexively.